### PR TITLE
Appointments - Back End Modifications

### PR DESCRIPTION
**Pull request by akroon3r**
Further development of the appointments feature required three new fields: service_id, citizen_name, contact_information

### CODE REVIEW

File: api/app/models/bookings/appointments.py [hunk 1]
```diff
@@ -21,13 +21,17 @@ class Appointment(Base):
 
     appointment_id = db.Column(db.Integer, primary_key=True, autoincrement=True, nullable=False)
     office_id = db.Column(db.Integer, db.ForeignKey("office.office_id"), nullable=False)
+    service_id = db.Column(db.Integer, db.ForeignKey("service.service_id"), nullable=False)
     start_time = db.Column(UtcDateTime, nullable=False)
     end_time = db.Column(UtcDateTime, nullable=False)
     category = db.Column(db.String(255), nullable=False)
```
Comment (sjrumsby) [inline api/app/models/bookings/appointments.py]: What is this field used for?

Comment (akroon3r) [reply api/app/models/bookings/appointments.py]: Screen Shot 2019-03-29 at 9 44 05 AM

Comment (akroon3r) [reply api/app/models/bookings/appointments.py]: Going off a slack conversation on this one.

Comment (sjrumsby) [reply api/app/models/bookings/appointments.py]: Yea, but why is category here, and a string?

**Pull request merged**